app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,21 @@
 module Api
   class AuthenticationsController < BaseController
+    def create_resource(_type, _id, data)
+      validate_auth_attrs(data)
+      klass = ::Authentication.class_from_request_data(data)
+      # TODO: Temporary validation - remove
+      raise 'type not currently supported' unless klass.respond_to?(:create_in_provider_queue)
+      manager_resource_id = parse_href(data['manager_resource']['href']).last
+      task_id = klass.create_in_provider_queue(manager_resource_id, data.except('type', 'manager_resource'))
+      MiqTask.find(task_id)
```

Comment:
@Fryguy @jameswnl assuming we should be returning the actual task from the ID that is returned?